scripts/roles/todd/vars/toddvars.yml

```diff
@@ -0,0 +1,10 @@
+docker_repo: 'deb https://apt.dockerproject.org/repo {{ ansible_distribution|lower }}-{{ ansible_distribution_release|lower }} main'

```

Comment:
If you place these vars in `todd/vars` for the role then they are not changeable from any other source of variable definitions. Is this what you want? Or would you want to allow them to be defined in either `host_vars` or `group_vars` for example? If you do not want them to be modified anywhere else then leave them within `todd/vars`.

Also add the following to the top of the file:
```
---
```